dynamo/plot/utils.py

```diff
@@ -556,12 +574,15 @@ def _matplotlib_points(
                     ]
                 )
         else:
+            main_debug("drawing legend")
             ax.legend(
                 handles=legend_elements,
-                bbox_to_anchor=(1.04, 1),
+                # bbox_to_anchor=(1.04, 1),
```

Comment:
this is needed, please uncomment it. this ensure the legend show up on the top right